src/markdown-pages/docs/first-steps/initialising-home-folder.md

```diff
@@ -0,0 +1,59 @@
+---
+path: "/docs/initialising-home-folder"
+title: "Initialising your local environment by using the CLI"
```

Comment:
CLI is command-line interface. Might not be a well-known abbreviation ?
So we need to spell it out a bit ? 

And the fact that CLI equates to `galasactl` might also be slightly confusing ? Not sure.

Reply:
Re the CLI, I have spelled it out in the top level topic of the section but added again in the overview section for this topic. I also added in info on galasactl to the top level topic.